CollapseLauncher/Classes/Properties/ArgumentParser.cs

```diff
@@ -207,6 +211,33 @@ public static void ParseMoveSteamArguments(params string[] args)
             var rootCommand = new RootCommand();
             rootCommand.AddCommand(command);
         }
+
+        private static void ParseStartGameArguments(params string[] args)
+        {
+            var gameOption = new Option<string>(new string[] { "--game", "-g" }, description: "Game") { IsRequired = true };
```

Comment:
Add possible options for this arguments, so that user is more informed on how to use it

Reply:
![image](https://github.com/CollapseLauncher/Collapse/assets/44784408/df0032fa-bb09-41ba-9288-a7ddf197378b)
Something like this?

Reply:
I think this one is adequate. Thanks!